net-irc/ngircd/files/ngircd.init-r2.d

```diff
@@ -0,0 +1,52 @@
+#!/sbin/openrc-run
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+extra_commands="configtest"
+extra_started_commands="reload"
+
+description="IRC server"
+description_configtest="Run ngircd's internal config check."
+description_reload="Reload the ngircd's configuration."
+
+: ${NGIRCD_CONFIGFILE:=/etc/ngircd/ngircd.conf}
+
+command="/usr/sbin/ngircd"
+command_args="${NGIRCD_OPTS} -f \"${NGIRCD_CONFIGFILE}\""
+command_args_foreground="-n"
+command_user="ngircd:ngircd"
+pidfile="${NGIRCD_PIDFILE:-/var/run/ngircd/ngircd.pid}"
```

Comment:
```suggestion
pidfile="${NGIRCD_PIDFILE:-/var/run/ngircd/${RV_SVCNAME}.pid}"
```

Reply:
pidfile is set in ngircd.conf, so it needs to be explicitly changed by the user anyways

Reply:
We can still pick a useful default.

Reply:
okay, done